treemap/index.js

```diff
@@ -0,0 +1,152 @@
+/**
+ * Bootstrapping code for line chart
+ */
+
+import * as d3 from 'd3';
+import * as gLegend from 'g-legend';
+import gChartframe from 'g-chartframe';
+import * as parseData from './parseData.js';
+import * as treemap from './treemap.js';
+
+const dataFile = 'treeData.csv';
+
+
+const sharedConfig = {
+    title: 'Title not yet added',
+    subtitle: 'Subtitle not yet added',
+    source: 'Source not yet added',
+};
+
+// const yMin = 0;// sets the minimum value on the yAxis
+// const yMax = 0;// sets the maximum value on the xAxis
```

Comment:
If there's no axes, it's probably worthwhile removing any commented code that references them.

Reply:
OK those now removed